File: ftp/php/read/ftp_read.php
#! /usr/bin/php
<?php
// ------------------------------------------------------------------
//	php/read/ftp_read.php
//
//					Oct/02/2012
//
// ------------------------------------------------------------------
$path="/var/www/data_base/common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);
//
include "curl_get.php";
include "text_manipulate.php";
// ------------------------------------------------------------------
print	"*** 開始 ***\n";

$url_in = 'ftp://scott:tiger@host_dbase/city/iwate.json';
print	"url_in = " .$url_in . "\n";

$json_string = curl_get_proc ($url_in);

# print	$json_string;
$dict_aa = json_decode ($json_string,true);
dict_display_proc ($dict_aa);

print	"*** 終了 ***\n";

// ------------------------------------------------------------------

// ------------------------------------------------------------------
?>

File: ftp/python/read/ftp_read.py
#! /usr/bin/python
# -*- coding: utf-8 -*-
#
#	read/ftp_read.py
#
#					Nov/30/2016
#
# ----------------------------------------------------------------
import os
import sys
import pycurl
import json
#
sys.path.append ('/var/www/data_base/common/python_common')
from text_manipulate import dict_display_proc
from curl_get import curl_get_proc
#
#
#
# ----------------------------------------------------------------
sys.stderr.write ("*** 開始 ***\n")
#
url_json = 'ftp://scott:tiger@host_dbase/city/iwate.json'
str_tmp = curl_get_proc (url_json)
#
dict_aa = json.loads (str (str_tmp,'UTF-8'))
dict_display_proc (dict_aa)
#
sys.stderr.write ("*** 終了 ***\n")
# ----------------------------------------------------------------
